test/e2etest/installer_test.go

```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
 
 	// Tests come here
 
+	// Test the plans
+	assertPlanListEndpoint(t, testConfig)
+
 	// Uninstall and cleanup the environment
 	runUninstallScript(testConfig.ConfigPath)
 }
 
+func assertPlanListEndpoint(t *testing.T, testConfig TestConfig) {
+	planUrl := "rest/api/latest/plan"
+	url := fmt.Sprintf("https://%s@%s.%s.%s/%s", credential, product, testConfig.EnvironmentName, domain, planUrl)
```

Comment:
Similar to https://github.com/atlassian-labs/data-center-terraform/pull/103#discussion_r786342371

Reply:
I replied to comment #103